cql2pgjson/src/main/java/org/folio/cql2pgjson/model/DbFkInfo.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.cql2pgjson.model;
+
+public class DbFkInfo {
```

Comment:
Can you rename it to a self-explanatory name, for example ForeignKey, avoiding the abbreviation Fk?
Is Db redundant because foreign keys are always in the db?
Is Info redundant because all classes with class variables contain info?

Reply:
I'll leave it as is.

Reply:
org.folio.rest.persist.ddlgen.Index parallels
org.folio.cql2pgjson.model.DbIndex

org.folio.rest.persist.ddlgen.ForeignKeys
parallels
org.folio.cql2pgjson.model.DbForeignKey
but not
org.folio.cql2pgjson.model.DbFkInfo

Can you explain why we want this inconsistency?